rerun_py/tests/unit/test_datafusion_tables.py

```diff
@@ -234,10 +234,48 @@ def test_partition_ordering(server_instance: tuple[subprocess.Popen[str], Datase
                         prior_timestamp = timestamp
 
 
-def test_url_generation(server_instance: tuple[subprocess.Popen[str], DatasetEntry]) -> None:
+def test_arrow_rb_reader(server_instance: tuple[subprocess.Popen[str], CatalogClient, DatasetEntry]) -> None:
```

Comment:
let maximise the legibility of that when it actually fails

```suggestion
def test_view_to_arrow_reader(server_instance: tuple[subprocess.Popen[str], CatalogClient, DatasetEntry]) -> None:
```

Reply:
This should test view, table, and table entry, but I will make it more descriptive.